TheOnlyThingIKnow.js

```diff
@@ -75,41 +85,49 @@ const twitchID = async () => {
         url: `https://api.twitch.tv/kraken/channels/${twitchChannelCode}/videos?sort=time`,
     });
 
-    let latestVOD = VODresponse.data.videos.reduce((latest, current) => {
-        if (!latest || Date.parse(current.created_at) > Date.parse(latest.created_at)) {
-            latest = current;
-        }
-        return latest;
+    let latestVOD = VODresponse.data.videos.reduce((latest, video) => {
+        return (Date.parse(video.created_at) > Date.parse(latest.created_at)) ? video : latest;
     });
     let VODID = latestVOD._id.substring(1);
 
     let mostRecent;
     while (true) { // This loop runs at the interval defined by lastFMPollInterval
         let lastFMResponse = await axios({
             method: "get",
-            url: `http://ws.audioscrobbler.com/2.0/?method=user.getrecenttracks&user=${lastFMUsername}&api_key=${secretData.lastFMAPI}&format=json&nowplaying="true"`,
+            url: `http://ws.audioscrobbler.com/2.0/?method=user.getrecenttracks&user=${lastFMUsername}&api_key=${secretData.lastFMAPI}&format=json`,
```

Comment:
Why not  just leave the now playing check to the URL? This will make the if statement where you check if the song is "nowplaying" redundant.

Reply:
Adding that parameter to the request doesn't seem to do anything. Last.fm still returns all recent tracks whether it is currently playing or not, so we'd still have to check for "nowplaying" in the response.